### ISSUE TITLE
Missing tray icon after unlock

### ISSUE THREAD

**Issue by gciatto-unibo** [cstayyab/WALC#40]
**Describe the bug**
The tray icon is missing after the system is unlocked, but WALC is still running, and there is no way to show it back.

**WALC Installation Type:** WALC 0.1.9(15), running on Ubuntu 20.04 LTS x64, installed through snap

**To Reproduce**
1. WALC is correctly started and running. The WA icon is correctly shown on the system tray
2. The user locks the system
3. The user unlocks the system by providing his/her password
4. The tray icon is gone, but WALC is still running
5. If WALC is minimised there is no way to show its window again

**Expected behavior**
The icon should not disappear if the system is (un)locked

**Did you try closing and then opening it back?:** Yes, that's the common solution: I must kill WALC and restart it

**System Information (please complete the following information):**
 - OS/Distro: Ubuntu 20.04 LTS, 64 bit

**Comment by cstayyab** [comment]
This issue not particularly related to WALC but in general with many apps including Signal-Desktop, Caprine and many more.

Following is the related upstream issue that must be fixed to get it working https://github.com/ubuntu/gnome-shell-extension-appindicator/issues/220

**Comment by validatedev** [comment]
Yeah, it happens to me also. But, my other applications are not affected by the bug. Thus, indeed there was a problem the app itself.

**Comment by Zzombiee2361** [comment]
Are these "other applications" you're talking about electron apps?

**Comment by validatedev** [comment]
> Are these "other applications" you're talking about electron apps?

i.e. there isn’t any problem with Discord and Telegram. Discord is an Electron app.

**Comment by cstayyab** [comment]
This problem is no longer reproducible at my end on Ubuntu 20.04. Can you please verify?

CC: @gciatto-unibo @validatedev
